File: TvJahnOrchesterApp.Api/TvJahnOrchesterApp.Contracts/Termine/Main/CreateTerminRequest.cs
using TvJahnOrchesterApp.Contracts.OrchestraMembers;

namespace TvJahnOrchesterApp.Contracts.Termine.Main
{
    public record CreateTerminRequest(string Name, int TerminArt, DateTime StartZeit, DateTime EndZeit, AdresseDto TreffPunkt, Guid[]? OrchestermitgliedIds, int[] Noten, int[] Uniform);
}

File: OrchesterApp.Api/OrchesterApp.Api/Frontend/src/app/termin/interfaces/create-termin-request.ts
export interface CreateTerminRequest {
  name: string,
  terminArt: number,
  startZeit: Date,
  endZeit: Date,
  straße: string,
  hausnummer: string,
  postleitzahl: string,
  stadt: string,
  zusatz?: string,
  latitude?: number,
  longitude?: number,
  noten: number[],
  uniform: number[],
  orchestermitgliedIds: string[],
  frist?: string,
  ersteWarnungVorFrist?: string
}